From c8880dcfc98f3643e8067d02591271c03729444e Mon Sep 17 00:00:00 2001
From: Holger Freyther <ich@tamarin.(none)>
Date: Sat, 13 Sep 2008 22:17:06 +0200
Subject: [PATCH] [binutils] Disable looking at the flags.....
     - These are for mixing ADV, PE and STD mode and I have no idea what this
       actually means. The libgcc symbols contain flags 0x10 and 0x11 but are
       all compiled with the same compiler and even EPSON's libgcc.a has
       object files with different flags.

---
 bfd/elf32-c33.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/bfd/elf32-c33.c b/bfd/elf32-c33.c
index ba5d1c8..c69e649 100644
--- a/bfd/elf32-c33.c
+++ b/bfd/elf32-c33.c
@@ -2003,6 +2003,7 @@ c33_elf_merge_private_bfd_data (ibfd, obfd)
 
 /* >>>>> ADDED D.Fujimoto 2007/10/01 link error for mixing core object files */
 	// get the flag from the object file
+#if 0
 	mode_flag = (unsigned char) (elf_elfheader(ibfd)->e_flags >> 24);
 
 	// get initial mode
@@ -2027,6 +2028,7 @@ c33_elf_merge_private_bfd_data (ibfd, obfd)
 
 	// set ELF e_flags bit31-28 (e_machine is set in elf.c)
 	elf_elfheader(obfd)->e_flags = initial_mode_flag << 24;
+#endif
 /* <<<<< ADDED D.Fujimoto 2007/10/01 link error for mixing core object files */
 
   if (! elf_flags_init (obfd))
-- 
1.5.4.3

